### PR TITLE
General Cleanup

### PR DESCRIPTION
**Pull request by StarlitGhost**
- prune `requirements.txt`
- remove old py2 compatibility stuff (`six`, `future`, `u''`)
- general module tidying (line spacing, line length, string formatting)

Probably pretty difficult to review, but there are no functional changes at all, and all modules load with no errors in my dev area (virtualenv created from scratch after the `requirements.txt` prune)

So, fingers crossed?

### CODE REVIEW

**Comment by StarlitGhost** [comment]
Also removed the `# -*- coding: utf-8 -*-` declarations, at Hubbe's suggestion

**Review by Heufneutje — APPROVED** [review]

File: desertbot/modules/commands/Sed.py [hunk 1]
```diff
@@ -158,12 +170,14 @@ def storeMessage(self, message, unmodified=True):
...
+        userList = self.unmodifiedMessages[message.replyTo]
+        userList.append(message)
+        self.messages[message.replyTo] = userList[-self.historySize:]
 
         if unmodified:
-            self.unmodifiedMessages[message.replyTo].append(message)
-            self.unmodifiedMessages[message.replyTo] = self.unmodifiedMessages[message.replyTo][-self.historySize:]
+            userList = self.unmodifiedMessages[message.replyTo]
+            userList.append(message)
+            self.unmodifiedMessagesi[message.replyTo] = userList[-self.historySize:]
```
Comment (HubbeKing) [inline desertbot/modules/commands/Sed.py]: Should be self.unmodifiedMessages[message.replyTo]

Comment (HubbeKing) [reply desertbot/modules/commands/Sed.py]: There's also some stuff broken with DataStore but that's unrelated to this, I'll fix that in master and update.

**Review by HubbeKing — APPROVED** [review]
I don't see anything else wrong and Travis doesn't show anything breaking anymore, so looks good. We'll fix things as they come up if there are typos hiding.

**Pull request merged**